cmd/run.go

```diff
@@ -92,6 +93,11 @@ func (r *RunConfig) dockerRunFlags() []cli.Flag {
 			Usage:       "Path of TLS client key to use (optional)",
 			Destination: &r.ClientKeyPath,
 		},
+		&cli.StringFlag{
+			Name:        "tls-server-name",
+			Usage:       "TLS server name to use for verification (optional)",
```

Comment:
```suggestion
			Usage:       "TLS server name to use for verification and SNI override (optional)",
```

(pedantic, not a big deal)